src/stats/utility.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module provides utility functions for the Calculator package in the stats project.
+"""
+
+from dataclasses import asdict, is_dataclass
+from typing import Any
+
+
+def convert_results(dataclass_instance: Any) -> dict:
+    """
+    Converts a dataclass instance to a dictionary.
+
+    Args:
+        dataclass_instance (dataclass): An instance of a dataclass.
+
+    Returns:
+        dict: A dictionary representation of the dataclass instance.
+    """
+    if not (
+        is_dataclass(dataclass_instance) and not isinstance(dataclass_instance, type)
+    ):
+        raise TypeError(
+            f"Expected a dataclass instance, got: {type(dataclass_instance)}"
```

Comment:
Who can see this? its if it's just for us (or other developers) its fine, but if UI end user will see it its will not be very helpful